metadata-ingestion/src/datahub/ingestion/source/redash.py

```diff
@@ -529,6 +536,16 @@ def _emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
                 # Continue producing MCE
                 dashboard_slug = dashboard_response["slug"]
                 dashboard_data = self.client.dashboard(dashboard_slug)
+
+                # Check dashboard queries datasource
+                for widget in dashboard_data["widgets"]:
+                    visualization = widget["visualization"]
+                    query_data = visualization["query"]
```

Comment:
Will query_data always be populated? 

If not, these lines can throw.

Reply:
https://github.com/datahub-project/datahub/pull/4616#discussion_r846512153